packages/now-routing-utils/test/index.spec.js

```diff
@@ -403,7 +411,82 @@ describe('normalizeRoutes', () => {
           },
           schemaPath: '#/items/additionalProperties',
         },
-      ],
+      ]
     );
   });
 });
+
+describe('getTransformedRoutes', () => {
+  test('should normalize nowConfig.routes', () => {
+    const nowConfig = { routes: [{ src: 'page', dest: '/page.html' }] };
+    const filePaths = [];
+    const actual = getTransformedRoutes({ nowConfig, filePaths });
+    const expected = normalizeRoutes(nowConfig.routes);
+    assert.deepEqual(actual, expected);
+    assertValid(actual.routes);
+  });
+
+  test('should normalize redirects before rewrites', () => {
+    const nowConfig = {
+      rewrites: [{ source: 'page', destination: '/page.html' }],
```

Comment:
Why does `source` not have a leading slash while `destination` does?

Reply:
I thought that normalize was supposed to add any missing forward slash since I definitely do this with existing now.json routes and it works today.

I modified this test to have a forward slash prefix.

Reply:
It only does this with `^`. You're thinking of `dest` that adds the leading slash automatically (in the proxy)

https://github.com/zeit/now/blob/64b24bde568895a16ff2c87a7d519af5a2ee7193/packages/now-routing-utils/src/index.ts#L71